### ISSUE TITLE
The generated docs should only report on the public APIs.

### ISSUE THREAD

**Issue by lijunle** [lijunle/Vsxmd#31]
It needs to reflect the generated DLL to get related information.

**Comment by micdah** [comment]
Agree, it is unfortunate to include `internal` classes and members

**Comment by lijunle** [comment]
Time past but I don't think anybody is going to improve this. People seems to be fine with the XML generated doc on this behavior. I am closing this issue. If there are more questions, please reply.

**Comment by spanglerco** [comment]
@lijunle, would you be interested in a pull request for this? I believe I have a working implementation.

The main limitation is that during build, dependencies are not available unless you wait to run Vsxmd on the output of a 'dotnet publish' build as a separate step.

**Comment by lijunle** [comment]
Reopen this issue.

@spanglerco It could be great if you have a working prototype. 👍

**Comment by spanglerco** [comment]
@lijunle Sure thing. I went ahead and created a PR. Feel free to merge or incorporate pieces of it as you wish.

**Comment by thomaslevesque** [comment]
Any news on this @lijunle? The PR has been sitting there for 3 months ;)

**Comment by vatsan-madhavan** [comment]
This is a great change! Hope it gets merged sometime soon.
